mcs/class/System/Test/System.Diagnostics/ProcessTest.cs

```diff
@@ -1146,6 +1148,7 @@ public void NonChildProcessWaitForExit ()
 		}
 
 		[Test]
+		[NUnit.Framework.Category ("MobileNotWorking")]
```

Comment:
Only this one fails with pretty odd message ` System.InvalidOperationException : Process has exited, so the requested information is not available.` Are you sure that's intended outcome?